include/linux/skbuff.h

```diff
@@ -950,8 +950,10 @@ enum {
 	TEMPESTA_TLS_SKB_TYPE_OFF	= 0,
 	TEMPESTA_TLS_SKB_TYPE_MAX	= 0x7F,
 	TEMPESTA_SKB_FLAG_OFF		= 7,
-	TEMPESTA_SKB_FLAG_CLEAR_MASK	= 0x01,
-	TEMPESTA_SKB_FLAG_MAX		= 0xFF,
+	TEMPESTA_SKB_FLAG_CLEAR_MASK	= 0x63,
```

Comment:
The define isn't used and `0x63` looks quite strange

Reply:
Fixed